### PR TITLE
Expected UI for register entity

### PR DESCRIPTION
**Pull request by rllola**
Print the expected UI for test

### CODE REVIEW

File: tests/manual_testvectors.json [hunk 1]
```diff
@@ -53,28 +53,31 @@
...
-                        "allow_entity_signed_nodes": true
-                    }
-                }
+                "untrusted_raw_value": "a362696458200000000000000000000000000000000000000000000000000000000000000000656e6f6465738258200000000000000000000000000000000000000000000000000000000000000001582000000000000000000000000000000000000000000000000000000000000000027819616c6c6f775f656e746974795f7369676e65645f6e6f646573f5"
             },
             "nonce": 123,
             "method": "registry.RegisterEntity"
         },
-        "encoded_tx": "o2JpZFggAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAASNlbm9kZXOCWCAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACFggAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAl4GWFsbG93X2VudGl0eV9zaWduZWRfbm9kZXP1",
+        "entity": {
```
Comment (jleni) [inline tests/manual_testvectors.json]: mmm.. I don't think Oasis test vectors will look like this... actually entity is supposed to come inside of untrusted_raw_value. 
The previous test generation script was doing that correctly, though, there might be a bug somewhere else.
I will review/debug a bit more the JS script but I prefer not to make this change. Hopefully, complete test vectors for all these new types will be available soon

File: tests/tools/template_testvectors.json [hunk 2]
```diff
@@ -50,22 +50,25 @@
         "kind": "RegisterEntity",
         "signature_context": "oasis-core/consensus: tx for chain test-chain-id",
         "tx": {
+            "fee": {
```
Comment (jleni) [inline tests/tools/template_testvectors.json]: Not sure we need to add fee here. If this is necessary for the test to work, this is indicating some kind of bug.

File: tests/tools/updateManualTestcase.js [hunk 3]
```diff
@@ -96,12 +84,13 @@ function toCBOR(root) {
     root = JSON.parse(JSON.stringify(root));
 
 // Now process the data and generate the correct cbor output
-    if ('entity' in root) {
-        tmp = fixFieldsForCBOR(root.entity);
+    if ('tx' in root) {
```
Comment (jleni) [inline tests/tools/updateManualTestcase.js]: I don't see why this change is necessary. I am probably missing something but it seems to be flipping the `if`.. 
if there is something else going on, it is probably relying on some kind of side effect, which is not so good I would say.

Comment (rllola) [reply tests/tools/updateManualTestcase.js]: It was linked to the structure I used in the template but it is outdated now.

File: tests/tools/template_testvectors.json (hunk 2 above)
Comment (rllola) [reply tests/tools/template_testvectors.json]: No it wasn't needed. No sure why I have added it but without fee test is passing.

File: tests/tools/template_testvectors.json [hunk 4]
```diff
@@ -50,22 +50,25 @@
...
                     "public_key": "0000000000000000000000000000000000000000000000000000000000000000"
                 },
-                "untrusted_raw_value": {
-                    "entity": {
-                        "id": "0000000000000000000000000000000000000000000000000000000000000123",
-                        "nodes": ["0000000000000000000000000000000000000000000000000000000000000008", "0000000000000000000000000000000000000000000000000000000000000009"],
-                        "allow_entity_signed_nodes": true
-                    }
-                }
+                "untrusted_raw_value": "a362696458200000000000000000000000000000000000000000000000000000000000000000656e6f6465738258200000000000000000000000000000000000000000000000000000000000000001582000000000000000000000000000000000000000000000000000000000000000027819616c6c6f775f656e746974795f7369676e65645f6e6f646573f5"
```
[the remark this answers is not in the document]
Comment (rllola) [reply tests/tools/template_testvectors.json]: I have reversed the changes like we discussed. The modification was done to fit the go structure (https://github.com/oasislabs/oasis-core/blob/754833e6500840ac1a163b491cd94e7e2441b7c3/go/common/crypto/signature/signature.go#L394-L401). However the generated tests vector will probably be with the entity in json and not the raw cbor value.
